README.md

```diff
@@ -85,24 +74,26 @@ export const counterReducer: ActionReducer<number> = (state: number = 0, action:
 }
 ```
 
-In your app's main module, import those reducers and use the `StoreModule.provideStore(reducers, initialState)` function to provide them to Angular's injector:
+In your app's main module, import those reducers and use the `StoreModule.provideStore(reducers)`
```

Comment:
Ah, then yes the example-app along with the linked angular-webpack2-starter will both be sources for AOT examples.